sig-gating-contracts/contracts/signerManager/SignerManagerProxyOwner.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import {INexeraIDSignerManager} from "./INexeraIDSignerManager.sol";
+
+/**
+ * @title SignerManagerProxyOwner
+ * @dev A contract to manage the NexeraID Signer Manager contract with role-based access control.
+ * SIGNER_MANAGER_CONTROLLER_ROLE controls changing the sigenrAddress on the NexeraID Signer Manager
+ * PAUSER_ROLE can only pause the contract (change the address to ONE_ADDRESS)
+ */
+contract SignerManagerProxyOwner is AccessControl {
+    INexeraIDSignerManager public signerManager;
+
+    // Roles
+    bytes32 public constant SIGNER_MANAGER_CONTROLLER_ROLE =
+        keccak256("SIGNER_MANAGER_CONTROLLER_ROLE");
+    bytes32 public constant PAUSER_ROLE = keccak256("PAUSER_ROLE");
+
+    // Non-existant address to pause signer
+    address private constant ONE_ADDRESS =
+        0x0000000000000000000000000000000000000001;
+
+    constructor(address signerManagerAddress) {
+        signerManager = INexeraIDSignerManager(signerManagerAddress);
+
+        _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _setupRole(SIGNER_MANAGER_CONTROLLER_ROLE, msg.sender);
+        _setupRole(PAUSER_ROLE, msg.sender);
+    }
+
+    // Function to set a new signer in the SignerManager contract
+    function setSigner(
+        address newSigner
+    ) external onlyRole(SIGNER_MANAGER_CONTROLLER_ROLE) {
+        signerManager.setSigner(newSigner);
+    }
+
+    // Function to transfer ownership of the SignerManager contract
+    function transferOwnership(
+        address newOwner
+    ) external onlyRole(SIGNER_MANAGER_CONTROLLER_ROLE) {
+        signerManager.transferOwnership(newOwner);
+    }
+
+    // Function to change the SIGNER_MANAGER_CONTROLLER_ROLE
+    function changeSignerManagerControllerRole(
```

Comment:
This can be removed, because `grantRole(DEFAULT_ADMIN_ROLE, account)` (exposed as `public` by `AccessControl`) can be used instead.